src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -17,95 +19,66 @@ public static class WellKnownRegexPatterns
-    };
-
-    public static IEnumerable<RegexPattern> HighConfidenceThirdPartySecurityModels { get; } = new List<RegexPattern>
-    {
-        new NpmAuthorKey(),
-        new SecretScanningSampleToken(),
-    };
+    public static IReadOnlyList<RegexPattern> UnclassifiedPotentialSecurityKeys { get; } =
+        CreateList(new UnclassifiedJwt(),
+                   new UrlCredentials(),
+                   new LooseSasSecret(),
+                   new OAuth2BearerToken(),
+                   new Unclassified32ByteBase64String(),
+                   new Unclassified64ByteBase64String(),
+                   new AadClientAppLegacyCredentials34(),      // SEC101/101 legacy generated passwords.
+                   new Pkcs12CertificatePrivateKeyBundle(),
+                   new Unclassified16ByteHexadecimalString()
+    );
+
+    public static IReadOnlyList<RegexPattern> HighConfidenceMicrosoftSecurityModels { get; } =
+        CreateList(new UnclassifiedLegacyCommonAnnotatedSecurityKey(),
+                   new AadClientAppIdentifiableCredentials(),
+                   new AzureFunctionIdentifiableKey(),
+                   new AzureSearchIdentifiableQueryKey(),
+                   new AzureSearchIdentifiableAdminKey(),
+                   new AzureRelayIdentifiableKey(),
+                   new AzureEventHubIdentifiableKey(),
+                   new AzureServiceBusIdentifiableKey(),
+                   new AzureIotHubIdentifiableKey(),
+                   new AzureIotDeviceIdentifiableKey(),
+                   new AzureIotDeviceProvisioningIdentifiableKey(),
+                   new AzureStorageAccountIdentifiableKey(),
+                   new AzureCosmosDBIdentifiableKey(),
+                   new AzureBatchIdentifiableKey(),
+                   new AzureMLWebServiceClassicIdentifiableKey(),
+                   new AzureApimIdentifiableDirectManagementKey(),
+                   new AzureApimIdentifiableSubscriptionKey(),
+                   new AzureApimIdentifiableGatewayKey(),
+                   new AzureApimIdentifiableRepositoryKey(),
+                   new AzureCacheForRedisIdentifiableKey(),
+                   new AzureContainerRegistryIdentifiableKey(),
+                   new NuGetApiKey(),
+                   new AdoLegacyPat(),                         // SEC101/102
+                   new AzureCosmosDBLegacyCredentials(),       // SEC101/104
+                   new AzureStorageAccountLegacyCredentials(), // SEC101/106
+                   new AzureMessagingLegacyCredentials(),
+                   new AzureDatabricksPat(),
+                   new AzureEventGridIdentifiableKey());
+
+    public static IReadOnlyList<RegexPattern> DataClassification { get; } =
+        CreateList(new IPv4(),
+                   new IPv6(),
+                   new Float(),
+                   new Integer(),
+                   new GuidValue());
+
+    public static IReadOnlyList<RegexPattern> HighConfidenceThirdPartySecurityModels { get; } =
```

Comment:
>HighConfidenceThirdPartySecurityModels [](http://example.com/codeflow?start=46&length=38)

I think we should zap this `ThirdParty` business, it's not helpful. Strictly speaking, MS doesn't own the databricks model either, so it's odd to have that one over in the other bucket. Let's just simplify to the high confidence bucket. We can add metadata later on the allocating company if it's helpful.

Reply:
I'm going to file one issue about improving these lists and keep these sorts of things out of this PR.